### DISCUSSION TITLE
v3.3.0-x86_64版，迅雷专家版方式添加网盘后播放的视频为720p

### DISCUSSION THREAD

**Discussion by LGuangming** [AlistGo/alist#2273] [General]
Openwrt上使用v3.3.0-x86_64版本，添加迅雷专家版成功后，之前视频分辨率一直没问题，最近播放的视频都是720p的分辨率，有人知道是为什么吗，不管是rclone还是webdav直接挂载都是以720p的方式播放。

**Comment by LGuangming** [comment]
没人有这个问题吗😂😂😂

**Comment by rakaki** [comment]
好像有人说过这个问题了，不是会员使用视频url的话就是720p

**Reply by 1738394450** [reply]
加一也是720p
